scripts/tests/test_deploy_sanity_gate.py

```
@@ -0,0 +1,134 @@
+    path.write_text(textwrap.dedent(content), encoding="utf-8")
+    path.chmod(path.stat().st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
+
+  @staticmethod
+  def _bash_path() -> str | None:
+    for candidate in (os.environ.get("TEST_BASH"), "/opt/homebrew/bin/bash", "/usr/local/bin/bash", "/bin/bash"):
+      if not candidate:
+        continue
+      path = Path(candidate)
+      if not path.is_file() or not os.access(path, os.X_OK):
+        continue
+      version = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)